l1-contracts/foundry.toml

```diff
@@ -1,6 +1,6 @@
 [profile.default]
 src = 'contracts'
-out = 'artifacts-forge'
+out = "out"
```

Comment:
why do we need this change?

Reply:
Unfortunately, slither does not work with `out = 'artifacts-forge'` even if to specify it explicitly via `--foundry-out-directory ./l1-contracts/artifacts-forge`